src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/NotYetReady/IfError.txt

```diff
@@ -0,0 +1,177 @@
+
+// ********** NON-ERROR PARAMETERS **********
+
+>> IfError(1, 2)
+1
+
+>> IfError(1, 2, 3)
+3
+
+>> IfError(1, 2, 3, 4)
+3
+
+>> IfError( 10, 11, 20, 21, 300 )
+300
+
+>> IfError(1, "error 1", 2, "error 2", "no error")
+"no error"
+
+>> IfError({a:1, b:"hello"}, {a:2, b:"good morning"}).b
+"hello"
+
+>> IfError(1, {a:1}, 2, {a:2}, {a:3}).a
+3
+
+>> IfError({a:{a1:true},b:"hello"}, {a:1,b:"error 1"}, {a:{a1:{a2:3}},b:"world"}, {a:1,b:"error 2"}, {a:1,b:"valid"}, {a:1,b:"error 3"}).a
+1

```

Comment:
There are three records with a=1; can you change the value of them so that the test can show which one of them is being returned?

Reply:
Changed to different values for a, to make it easier to understand the return value